src/Kconfig.coap

```diff
@@ -40,12 +40,42 @@ config GOLIOTH_MAX_NUM_OBSERVATIONS
     help
         The maximum number of CoAP paths which can be simultaneously observed.
 
-config GOLIOTH_BLOCKWISE_UPLOAD_BLOCK_SIZE
-    int "Golioth blockwise uploads block size"
-    default 1024
+choice
+    prompt "Golioth blockwise upload: Max block size"
+
+    config UP_BLOCKSIZE_1024
+        bool "1024"
+    config UP_BLOCKSIZE_512
+        bool "512"
+    config UP_BLOCKSIZE_256
+        bool "256"
+    config UP_BLOCKSIZE_128
+        bool "128"
+    config UP_BLOCKSIZE_64
+        bool "64"
+    config UP_BLOCKSIZE_32
+        bool "32"
+    config UP_BLOCKSIZE_16
+        bool "16"
+
     help
         Block size in Blockwise Uploads.
```

Comment:
Shoudn't `help` be just after `prompt`, before any choices listed with `config`?